standard/normative-references.md

```diff
@@ -10,4 +10,6 @@ ISO/IEC 2382, *Information technology — Vocabulary*.
 
 ISO/IEC 60559:2020, *Information technology — Microprocessor Systems — Floating-Point arithmetic*
 
+*.NET API Documentation*, https://learn.microsoft.com/en-us/dotnet/api
```

Comment:
For C# 8, we should link to the .NET Core 3.1 version of the APIs. We should also remove the locale:

```suggestion
*.NET API Documentation*, https://learn.microsoft.com/dotnet/api/?view=netcore-3.1
```